tests/unit/lib/thrift_pool_tests.py

```diff
@@ -4,8 +4,7 @@
 
 from unittest import mock
 
-import gevent.queue
-
+from gevent import queue as gevent_queue
```

Comment:
A couple more changes:
* remove `import queue` on L1
* change `gevent.queue` to `gevent_queue` on L130 (but would then conflict with [this PR](https://github.com/reddit/baseplate.py/pull/656/files#diff-3de570a9b43ce8e7774ac672c91e3f49f999a4d88310d9afb12bd997a840a6fdL7))

Reply:
I flipped this around to remove my import change and fixed the test to use `gevent.queue`.